lib/librato-sidekiq/middleware.rb

```diff
@@ -62,30 +62,38 @@ def call(worker_instance, msg, queue, redis_pool = nil)
         return result unless enabled
         # puts "#{worker_instance} #{queue}"
 
+        # From the docs (https://github.com/mperham/sidekiq/wiki/Job-Format):
+        # Note that enqueued_at isn't added to the payload for scheduled jobs.
+        # They get the enqueued_at field when they are pushed onto a queue.
+        enqueued_at = msg['enqueued_at'] || start_time
```

Comment:
Will the time format of `msg['enqueued_at']` match the format of `start_time`?

Reply:
`msg['enqueued_at']` comes as the number of seconds since epoch (Float)
`start_time` is `Time`, which accepts operations like `Time - Float`. 

I am happy to cast it, I am just waiting to resolve the above comment so we can decide the direction we want to take